Deprecate identity provider feature (#3984)
replicatedhq/replicated-docs: docs/enterprise/auth-identity-provider.md

# Use an identity provider for user access (Deprecated)

:::note

> Replicated deprecated the identity service. This feature works only in clusters created by Replicated kURL. Replicated will not extend it to other install methods such as KOTS existing cluster or Embedded Cluster, and does not plan further development.

Reviewer: ```suggestion
The Replicated identity service feature is deprecated. This feature works only in clusters created by Replicated kURL. Replicated will not extend the identity service to other installation methods such as KOTS existing cluster or Embedded Cluster, and does not plan further development.
```
Recommend the Vale-despised passive voice in this case (if for no other reason than to avoid the rhyming 😄 )
